src/main/java/org/web3j/console/project/utills/ProgressCounter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project.utills;
+
+import java.io.IOException;
+
+public class ProgressCounter {
+    public void processing(boolean loading, String message) {
+        Thread th =
+                new Thread(
+                        () -> {
+                            String anim = "|/―\\";
+                            try {
+                                System.out.write("\r|".getBytes());
+                                int current = 0;
+                                while (loading) {
```

Comment:
How does this break?

Reply:
on error or on success system.exit is called.